IdentityCore/src/oauth2/account/MSIDAccountIdentifier.m

```diff
@@ -161,4 +164,38 @@ - (BOOL)isEqualToItem:(MSIDAccountIdentifier *)account
     return result;
 }
 
+#pragma mark - MSIDJsonSerializable
+
+- (instancetype)initWithJSONDictionary:(NSDictionary *)json error:(NSError **)error
```

Comment:
Will this also be used for interactive operations? We don't technically require any identifier to be present for interactive operations, wanted to make sure we don't fail then.

Reply:
fixed by ignoring MSIDAccountIdentifier errors in interactive request.